rs/nns/governance/tests/governance.rs

```diff
@@ -14045,6 +14041,57 @@ async fn test_follow_public_neuron_succeeds() {
 
     assert!(result.is_ok());
 
+    // Make a proposal to have neuron 3 follow neuron 1.
+    gov.make_proposal(
+        &NeuronId { id: MANAGER_ID },
+        &principal(MANAGER_ID),
+        &Proposal {
+            title: Some("N3 follow N1".to_string()),
+            action: Some(proposal::Action::ManageNeuron(Box::new(ManageNeuron {
+                neuron_id_or_subaccount: Some(NeuronIdOrSubaccount::NeuronId(NeuronId { id: 3 })),
+                id: None,
+                command: Some(manage_neuron::Command::Follow(manage_neuron::Follow {
+                    topic: Topic::ExchangeRate as i32,
```

Comment:
I think this is obsolete now? I guess this only becomes a problem if we try to get rid of it, or something, so not a big deal, but since it is pretty easy to change this aspect of the test, why not use a value that is more useful in practice?